docs: tw924-customization (#3433)
OffchainLabs/arbitrum-docs: docs/launch-arbitrum-chain/overview/introduction.mdx

Author: @leeederek @asomani-ocl

sidebar_position: 1

> ---

Reviewer: One comment overall - there are some features (e.g. block time, dynamic pricing) which are repeated across sections. Is there a better way to showcase them than repeating them multiple times? Maybe we decide the best possible place for them and keep it there or if we repeat them then we sort of cross-reference to an earlier description?

> Arbitrum chains give you flexibility and control without the constraint of running your own Layer 1 blockchain. Instead of bootstrapping and subsidizing your own validator set, your chain [anchors its security and finality to Ethereum](/how-arbitrum-works/deep-dives/transaction-lifecycle.mdx), turning security into a variable, usage-based expense.

Reviewer 2: ```suggestion
Arbitrum chains give you flexibility and control without the constraint of running your own <a data-quicklook-from="layer-1-l1">Layer 1</a> blockchain. Instead of bootstrapping and subsidizing your own validator set, your chain [anchors its security and finality to Ethereum](/how-arbitrum-works/deep-dives/transaction-lifecycle.mdx), turning security into a variable, usage-based expense.
```

> An Arbitrum chain gives you the flexibility and control enterprises actually want—without the burden of running your own Layer 1. Instead of bootstrapping and subsidizing your own validator set, your chain [anchors its security and finality to Ethereum](/how-arbitrum-works/deep-dives/transaction-lifecycle.mdx), turning security into a variable, usage-based expense.

Reviewer 2: ```suggestion
Arbitrum chains give you flexibility and control without the constraint of running your own Layer 1 blockchain. Instead of bootstrapping and subsidizing your own validator set, your chain [anchors its security and finality to Ethereum](/how-arbitrum-works/deep-dives/transaction-lifecycle.mdx), turning security into a variable, usage-based expense.
```

> An Arbitrum chain gives you the flexibility and control enterprises actually want—without the burden of running your own Layer 1. Instead of bootstrapping and subsidizing a validator set (a fixed cost that runs into the billions annually across major L1s), your chain anchors its security and finality to Ethereum's nearly 1 million validators and $78B in staked ETH, turning security into a variable, usage-based expense.

Reviewer 3: I believe you should cite a source here if you can.

> That economic model is the core reason to choose Arbitrum: your business captures the fee revenue and priority-access value that L1s hand off to validators, achieving 90–98% operating margins because costs scale with usage rather than perpetual subsidies. You also control the economics directly—custom gas tokens, fee policy, and revenue capture—while getting institution-grade settlement: sub-second soft finality for a responsive user experience, configurable hard finality on Ethereum in minutes, and native withdrawals in as little as 15 minutes, backed by a clean, externally legible counterparty-risk story for your risk, audit, and compliance stakeholders.

Reviewer 3: Same comment as above: a source would be great here.

> That economic model is the core reason to choose Arbitrum: your business captures the fee revenue and priority-access value that L1s hand off to validators, because costs scale with usage rather than perpetual subsidies. You also control the economics directly—[custom gas tokens](/launch-arbitrum-chain/chain-config/costs/custom-gas-token-rollup.mdx), [fee policy](/launch-arbitrum-chain/chain-config/costs/fee-management.mdx), and revenue capture—while getting institution-grade settlement: sub-second soft finality for a responsive user experience, [configurable hard finality on Ethereum in minutes](/launch-arbitrum-chain/chain-config/sequencer/chain-finality.mdx), and [native withdrawals in as little as 15 minutes](/launch-arbitrum-chain/chain-config/validation/fast-withdrawals.mdx), backed by a clean, externally legible counterparty-risk story for your risk, audit, and compliance stakeholders.

Reviewer 2: "Because costs scale with usage rather than perpetual subsidies. "
I don't understand this sentence.

Author: I gave a stab at trying to reword this to fit better with all audiences.

Beyond economics, an Arbitrum chain lets you launch fast and customize deeply while offloading the operational complexity of running blockchain infrastructure. You get day-one configuration over high, fully tunable throughput and block times (as low as 100ms), data availability (Rollup, AnyTrust, or external DA), sequencing and MEV rules, KYC/AML and permissioning, privacy, precompiles, governance, and multi-prover settlement—and you automatically inherit every future Ethereum and Arbitrum upgrade, including Stylus, without custom engineering.

> Critically, your chain isn't siloed: it plugs directly into Ethereum's deep liquidity and the Arbitrum ecosystem (Arbitrum One alone secures over $16B with 1,000+ apps and millions of users). And you can de-risk your go-to-market with a phased "launch-and-migrate" path—prove your product on the shared, liquid Arbitrum One, then graduate to your own dedicated Arbitrum chain as a seamless continuation on the same stack, not a costly replatforming.

Reviewer 3: Another place to insert a source for the figure you cite!

> Beyond economics, an Arbitrum chain lets you launch fast and customize deeply while offloading the operational complexity of running blockchain infrastructure. You get day-one configuration over high, fully tunable throughput and [block times (as low as 100ms)](/launch-arbitrum-chain/chain-config/sequencer/sequencer-timing-adjustments.mdx), [data availability (Rollup, AnyTrust, or external DA)](/launch-arbitrum-chain/chain-config/data-availability/config-data-availability.mdx), [sequencing and MEV rules](/how-arbitrum-works/timeboost/gentle-introduction.mdx), KYC/AML and permissioning, privacy, [precompiles](/launch-arbitrum-chain/protocol-hacks/precompiles.mdx), governance, and multi-prover settlement—and you automatically inherit every future Ethereum and Arbitrum upgrade, including [Stylus](/stylus/gentle-introduction.mdx), without custom engineering.

Reviewer 2: "Beyond economics, an Arbitrum chain lets you launch fast and customize deeply while offloading the operational complexity of running blockchain infrastructure."
Does it? They still need validator nodes and Sequencers.

| Feature                                                       | Your benefit                                                                                                                                                                                            | User benefit                                                                                                                                                          | Guide                                                                                                            |
| ------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |

> | Tune block time                                               | You own the speed-versus-cost tradeoff. Faster blocks let you position the chain as a premium, low-latency venue (trading, gaming, payments) without waiting on a third party to change infrastructure. | Near-instant confirmations make the app feel like a familiar web experience rather than a slow onchain one, reducing the “did my transaction go through?” hesitation. | [Configure chain finality](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/sequencer/chain-finality) |

Reviewer: We might need a better page here since this guide doesnt have anything related to block time as such. Maybe we can add something about block time on this page and use that - https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/sequencer/sequencer-timing-adjustments

> | Configure delayed inbox and finality for deposits/withdrawals | You match certainty guarantees to your actual risk profile instead of over-provisioning, so you avoid paying for stronger finality than your product needs.                                             | Predictable, well-defined settlement means users know exactly when funds and actions are final—important for anyone moving real value.                                | [Configure chain finality](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/sequencer/chain-finality) |

Reviewer: this point feels unclear to me. as a new user i don't know what "delayed inbox" means. could we name the feature around what it does, something like "configurable deposit finality"? makes it easier to understand.
the "your benefit" line is also not very clear, mainly "over-provisioning", not clear what we are referring to as over-provisioning here or what the actual tradeoff is.

also this row and the next one (fast withdrawals) both talk about withdrawals, which is confusing. cleaner split might be to make this row only about deposits and the next only about withdrawals. that removes the overlap and makes the rename to "deposit finality" fit naturally.

> | Configure deposit finality (delayed inbox)                 | You match certainty guarantees to your actual risk profile to avoid paying for stronger finality than your product needs.                                                                               | Predictable, well-defined settlement means users know exactly when funds and actions are final—important for anyone moving real value.                                | [Configure chain finality](/launch-arbitrum-chain/chain-config/sequencer/chain-finality.mdx)                                                                                                                          |

Reviewer 3: This is fine and OK to mention but the linked page doesn't go into detail about how to actually configure the parameter you need to configure to reduce deposit wait times. 

I suggest you include the parameter and update the linked doc to describe how to do it and the tradeoffs. A quick Claude search showed me that you care about delayedSequencerConfigs and the time that the chain considers something on its parent chain "final"

Author: sequencer-timing-adjustments covers this—albeit in part—I'm expanding to include all 6 flags (instead of 3).

| Enable fast withdrawals to reduce withdrawal finality time    | Fewer support tickets and complaints about locked-up capital, and a more competitive bridging story when users compare your chain to alternatives.                                                      | Users get their funds in minutes instead of waiting the full challenge window, dramatically lowering the friction of exiting the chain.                               | [Fast withdrawals](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/validation/fast-withdrawals)      |

> <VanillaAdmonition type="note">The 100ms figure is an optional lower bound on block time, not the default. Most chains run at the 250ms default; 100ms is available when you opt into it.</VanillaAdmonition>

Reviewer: In this note should we also add about the fast withdrawals not being the default and by default withdrawal time would be ~6.4 days?

> | Configure deposit finality (delayed inbox)                 | You match certainty guarantees to your actual risk profile to avoid paying for stronger finality than your product needs. Adjust the time before the chain processes a deposit.                         | Predictable, well-defined settlement means users know exactly when funds and actions are final—important for anyone moving real value.                                | [Configure chain finality](/launch-arbitrum-chain/chain-config/sequencer/chain-finality.mdx)                                                                                                                          |

Reviewer 2: ```suggestion
| Configure deposit finality (<a data-quicklook-from="delayed-inbox">delayed inbox</a>)                 | You match certainty guarantees to your actual risk profile to avoid paying for stronger finality than your product needs. Adjust the time before the chain processes a deposit.                         | Predictable, well-defined settlement means users know exactly when funds and actions are final—important for anyone moving real value.                                | [Configure chain finality](/launch-arbitrum-chain/chain-config/sequencer/chain-finality.mdx)                                                                                                                          |
```

> | Enable fast withdrawals to reduce withdrawal finality time | Fewer support tickets and complaints about locked-up capital, and a more competitive bridging story when users compare your chain to alternatives.                                                      | Users get their funds in minutes instead of waiting the full challenge window, dramatically lowering the friction of exiting the chain.                               | [Fast withdrawals](/launch-arbitrum-chain/chain-config/validation/fast-withdrawals.mdx)                                                                                                                               |

Reviewer 2: ```suggestion
| Enable <a data-quicklook-from="fast-withdrawals">fast withdrawals</a> to reduce withdrawal finality time | Fewer support tickets and complaints about locked-up capital, and a more competitive bridging story when users compare your chain to alternatives.                                                      | Users get their funds in minutes instead of waiting the full challenge window, dramatically lowering the friction of exiting the chain.                               | [Fast withdrawals](/launch-arbitrum-chain/chain-config/validation/fast-withdrawals.mdx)                                                                                                                               |
```

| Manage the fee parameters that govern what uses pay and fee distribution        | Direct control over cost recovery and revenue, so you can tune the chain's economics to be self-sustaining or subsidized as your business model requires.                   | Transparent, deliberately set fees rather than opaque or volatile costs, which builds trust in the pricing.                | [Fee management](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/fee-management)                                                                                                                                                           |

> | Configure native mint/burn behavior for the gas token, and tune dynamic pricing | Automating supply and fee handling protects your margins during demand swings and reduces manual treasury operations.                                                       | More stable, predictable costs even when the network is busy, so users aren't surprised by fee spikes.                     | - [Native mint and burn](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/configure-native-mint-burn) <br /> - [Dynamic pricing](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/dynamic-pricing)                         |

Reviewer: these two should be separate points, they solve different problems. 
1. mint/burn is about the supply/bridging model for the native gas token: lets teams use existing omnichain tokens as gas without a separate bridging mechanism. 
2. dynamic pricing is about how gas price responds to demand, giving control over how the chain behaves under load. 
bundling them might not give clear picture of what we are really offering here.

### Throughput and latency

Handle higher volumes and faster response times for products that cannot degrade during periods of peak demand.

| Feature                                                                                           | Your benefit                                                                                                                                                    | User benefit                                                                                                                           | Guide                                                                                                          |
| ------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |

> | Reserve dedicated throughput so your chain does not compete for computation and storage resources | Guaranteed capacity means you can make performance commitments (effectively an SLA) to partners and customers without worrying about noisy-neighbor contention. | Consistent performance during peak events—launches, drops, market volatility—instead of degraded speed exactly when demand is highest. | [Launch overview (dedicated throughput)](https://docs.arbitrum.io/launch-arbitrum-chain/overview/introduction) |

Reviewer: ```suggestion
| Dedicated throughput so your chain does not compete for computation and storage resources | Guaranteed capacity means you can make performance commitments (effectively an SLA) to partners and customers without worrying about noisy-neighbor contention. | Consistent performance during peak events—launches, drops, market volatility—instead of degraded speed exactly when demand is highest. | [Launch overview (dedicated throughput)](https://docs.arbitrum.io/launch-arbitrum-chain/overview/introduction) |
```

> | Set the gas target and block gas limit to match your expected load                                | You right-size capacity to your cost budget and build in headroom for growth, avoiding both overspend and under-provisioning.                                   | Fewer failed or delayed transactions at peak, because the chain has enough room to absorb bursts.                                      | [Gas target guidance](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/gas-target)            |

Reviewer: For the benefit, i guess we can make it a bit more clear on what does overspend here mean. I guess we wanted to talk about the stateroot growth and related infra costs. So maybe we can make it clear to something like avoiding high infra costs vs higher costs for end users (due to lower gas target setting).

Also we mentioned block gas limit, so we can add this link as well - https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/gas-optimization#block-gas-limit

> | Tune the block time and gas trade-offs for your performance profile                               | You optimize the cost-versus-speed balance for your specific workload rather than accepting a one-size-fits-all default.                                        | A responsive app that stays fast without passing along unnecessary fee increases.                                                      | [Dynamic pricing](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/costs/dynamic-pricing)           |

Reviewer: Maybe we just focus on dynamic pricing here instead of both block time and dynamic pricing. Since both of them solve for different problems, the benefits aren't coming out clearly.
For dynamic pricing - benefit is that it gives the chain operator control over how the chain gas price responds to demand. 

We can make blocktime a separate point if we want to add it here as well.

Author: Since we are splitting the above Mint/burn and dynamic pricing, I'll remove this point, since it would be duplicative.

| Run permissioned validators to vet and restrict who participates in validation—useful for enterprise or regulated environments (for example, KYC for validators) | You can meet regulatory and legal requirements, reduce compliance risk, and make the chain viable for enterprise and institutional partners who can't operate on a fully open infrastructure. | Assurance that the chain operates within a vetted, accountable set of participants — a prerequisite for many regulated financial and enterprise products. | [Validation and BoLD](/launch-arbitrum-chain/chain-config/validation/bold.mdx)                                    |

> | Restrict who can read chain data by keeping data off the public L1 with an AnyTrust data availability committee                                                  | You keep sensitive business and user data out of a fully public ledger, satisfying privacy obligations and protecting competitive information.                                                | Greater confidentiality around their activity and data than a fully public chain would offer.                                                             | [Configure data availability](/launch-arbitrum-chain/chain-config/data-availability/config-data-availability.mdx) |

Reviewer 2: ```suggestion
| Restrict who can read chain data by keeping data off the public <a data-quicklook-from="L1">L1</a> with an AnyTrust <a data-quicklook-from="data-availability-committee-dac">data availability committee</a>                                                  | You keep sensitive business and user data out of a fully public ledger, satisfying privacy obligations and protecting competitive information.                                                | Greater confidentiality around their activity and data than a fully public chain would offer.                                                             | [Configure data availability](/launch-arbitrum-chain/chain-config/data-availability/config-data-availability.mdx) |
```

> | Move to permissionless validation later via BoLD when you are ready to decentralize                                                                              | You can launch with tight control and decentralize on your own timeline — no re-platforming required as your product and risk tolerance mature.                                               | A credible path to stronger trustlessness and censorship resistance over time, rather than being locked into a permissioned model forever.                | [Validation and BoLD](/launch-arbitrum-chain/chain-config/validation/bold.mdx)                                    |

Reviewer 2: ```suggestion
| Move to permissionless validation later via <a data-quicklook-from="bold">BoLD</a> when you are ready to decentralize                                                                              | You can launch with tight control and decentralize on your own timeline — no re-platforming required as your product and risk tolerance mature.                                               | A credible path to stronger trustlessness and censorship resistance over time, rather than being locked into a permissioned model forever.                | [Validation and BoLD](/launch-arbitrum-chain/chain-config/validation/bold.mdx)                                    |
```

| --------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- |
| Keep the default First-Come, First-Serve (FCFS) ordering, which protects users from front-running and sandwich MEV                | Fair, simple ordering keeps fast block times and avoids the reputational cost of a chain seen as hostile to ordinary users. | Built-in protection from front-running and sandwich attacks, so users aren't quietly taxed by MEV extractors on every trade.                 | [How the Sequencer works](https://docs.arbitrum.io/how-arbitrum-works/deep-dives/sequencer)                |

> | Enable Timeboost to auction an express lane, letting the chain owner capture MEV while preserving fair ordering for everyone else | You capture MEV as a revenue stream for the chain rather than leaking it to external searchers.                             | Non-express transactions keep their fair-ordering protections, while users who genuinely need priority have a transparent way to pay for it. | [Timeboost configuration](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/sequencer/timeboost) |

Reviewer: should we also add a line item for PGA with "coming soon". In case it helps to showcase more customizations.

Author: It's our practice of not putting unpublished features in the docs.

> | Keep the default First-Come, First-Serve (FCFS) ordering, which protects users from front-running and sandwich MEV                | Fair, simple ordering keeps fast block times and avoids the reputational cost of a chain seen as hostile to ordinary users. | Built-in protection from front-running and sandwich attacks, so users aren't quietly taxed by MEV extractors on every trade.                 | [How the Sequencer works](/how-arbitrum-works/deep-dives/sequencer.mdx)                |

Reviewer 3: The protection for users from front-running and sandwich MEV comes for free regardless of the txn ordering policy. Instead, it comes from the fact that the information is private until sequenced. 

I would suggest changing this to read something like: "Keep the default FCFS ordering for intuitive, simple ordering that the world's largest exchanges uses for trading (e.g. NYSE)". 

In other words, the benefit of FCFS is from its simplicity and the fact that its used everywhere normally. It will also favor more teams accustomed to trading in the real world (not on a blockchain)

> | Enable Timeboost to auction an express lane, letting the chain owner capture MEV while preserving fair ordering for everyone else | You capture MEV as a revenue stream for the chain rather than leaking it to external searchers.                             | Non-express transactions keep their fair-ordering protections, while users who genuinely need priority have a transparent way to pay for it. | [Timeboost configuration](/launch-arbitrum-chain/chain-config/sequencer/timeboost.mdx) |

Reviewer 3: We are killing Timeboost, suggest replacing this with Priority Gas Auctions (PGA)!

```suggestion
| Enable Priority Gas Auctions (PGA), allowing transactions to include a tip (called a Priority Fee) to express ordering preferences (higher tip, earlier in the block). This enables the chain owner capture MEV while preserving fair ordering for everyone else | You capture MEV as a revenue stream for the chain rather than leaking it to entirely external arbitrageurs.                             | Protections are in place to ensure non- or low-priority fee paying transactions are included in a timely manner during volatility. | TODO PGA DOCS |
```

Reviewer 3: PGA also comes with the free frontrunning and sandwich mev protections you described earlier.

Author: This isn't implemented yet—we will hold off

> | Keep the default FCFS ordering for intuitive, simple ordering that the world's largest exchanges uses for trading                 | Fair, simple ordering keeps fast block times and avoids the reputational cost of a chain seen as hostile to ordinary users. | Built-in protection from front-running and sandwich attacks, so users aren't quietly taxed by MEV extractors on every trade.                 | [How the Sequencer works](/how-arbitrum-works/deep-dives/sequencer.mdx)                |

Reviewer 2: ```suggestion
| Keep the default FCFS ordering for intuitive, simple ordering that the world's largest exchanges use for trading                 | Fair, simple ordering keeps fast block times and avoids the reputational cost of a chain seen as hostile to ordinary users. | Built-in protection from front-running and sandwich attacks, so users aren't quietly taxed by MEV extractors on every trade.                 | [How the Sequencer works](/how-arbitrum-works/deep-dives/sequencer.mdx)                |
```

| Define the chain-owner role and access controls that govern upgrades and administration, and plan a path toward progressive decentralization | You retain the control needed for upgrades and incident response early on, then deliberately decentralize as the chain matures—balancing agility with credibility. | Clear accountability for who can change the chain, plus a visible path toward stronger, more decentralized guarantees.                                       | [Ownership and access control](https://docs.arbitrum.io/launch-arbitrum-chain/operate/ownership-and-access)                           |

> | Choose your data availability model—Rollup, AnyTrust, Alt-DA—to trade off cost against transparency and resilience                           | You dial the cost-versus-security balance directly; AnyTrust can cut data costs substantially, while Rollup maximizes security and transparency.                   | Lower fees when you choose AnyTrust, or maximum security and Ethereum-grade transparency when you choose Rollup—matched to what your product actually needs. | [Configure data availability](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/data-availability/config-data-availability) |

Reviewer: ```suggestion
| Choose your data availability model—Rollup, AnyTrust, Alt-DA—to trade off cost against transparency and resilience                           | You dial the cost-versus-security balance directly; AnyTrust can cut data costs substantially, while Rollup maximizes security and transparency.                   | Lower fees when you choose AnyTrust, or maximum security and Ethereum-grade transparency when you choose Rollup. | [Configure data availability](https://docs.arbitrum.io/launch-arbitrum-chain/chain-config/data-availability/config-data-availability) |
```

> | Choose your data availability model—Rollup, AnyTrust, Alt-DA—to trade off cost against transparency and resilience                           | You dial the cost-versus-security balance directly; AnyTrust can cut data costs substantially, while Rollup maximizes security and transparency.                   | Lower fees when you choose AnyTrust, or maximum security and Ethereum-grade transparency when you choose Rollup        | [Configure data availability](/launch-arbitrum-chain/chain-config/data-availability/config-data-availability.mdx) |

Reviewer 2: ```suggestion
| Choose your data availability model—Rollup, AnyTrust, <a data-quicklook-from="alt-da">Alt-DA</a>—to trade off cost against transparency and resilience                           | You dial the cost-versus-security balance directly; AnyTrust can cut data costs substantially, while Rollup maximizes security and transparency.                   | Lower fees when you choose AnyTrust, or maximum security and Ethereum-grade transparency when you choose Rollup        | [Configure data availability](/launch-arbitrum-chain/chain-config/data-availability/config-data-availability.mdx) |
```

## What problem do Arbitrum chains solve?

Arbitrum chains are dedicated chains built with Arbitrum technology. Teams can configure execution, fee models, governance, data availability, validation, and other chain parameters for their application or business requirements.

The Ethereum ecosystem is supported by a **decentralized network of nodes** that each run Ethereum's Layer 1 (L1) client software. Ethereum's block space is in high demand, so users are often stuck waiting for the network to become less congested (and thus, less expensive).

Arbitrum's protocols address this challenge by offloading some of the Ethereum network's heavy lifting to **another decentralized network of nodes** that support the Arbitrum stack (Arbitrum chains).

## How do Arbitrum chains help the Ethereum ecosystem?

> Arbitrum helps Ethereum move towards a **multi-chain future**. This is valuable for the following reasons:

Reviewer 3: Something Id mention here is that you get the best of both worlds by choosing our technology:
- you get Ethereum-grade security and interoperability, so your assets, users, and buidlers flow to/from Ethereum seamlessly and safely
- you get your own little island to customize/tune for your specific use case however you want - the world's your oyster!

Author: I'll come up with something—the second line is something we've largely gotten away from, but I think it's worth stating in a slightly different manner.

It depends on your definition of "app chain". Arbitrum chains can be used as application-specific chains (often referred to as "app chains" or "appchains"). But **they aren't just for apps**. They're for **hosting EVM-compatible smart contracts using self-managed infrastructure that isolates compute resources away from Arbitrum's public L2 chains** based on your unique needs.

- You can use your Arbitrum chain to host the smart contracts that support one app, two apps, an ecosystem of apps, or no apps at all.

> - Ethereum-grade security and interoperability, so your assets, users, and buidlers flow to/from Ethereum seamlessly and safely.

Reviewer 2: ```suggestion
- Ethereum-grade security and interoperability, so your assets, users, and builders flow to/from Ethereum seamlessly and safely.
```
